components/automate-gateway/handler/authz_test.go

```diff
@@ -36,8 +31,8 @@ import (
 	_ "github.com/chef/automate/components/automate-gateway/api/compliance/profiles"
 )
 
-func TestIntrospectAllV1(t *testing.T) {
-	authzSrv, _, s, hdlr := testServerAndHandler(t)
+func TestIntrospectAllV2(t *testing.T) {
```

Comment:
Is it necessary to keep `V2` in the name?

Reply:
No; good catch.